test/src/common/harnessIo.c

```diff
@@ -0,0 +1,75 @@
+/***********************************************************************************************************************************
+Harness for Testing
+***********************************************************************************************************************************/
+#include "build.auto.h"
+
+#include <stdlib.h>
+#include <unistd.h>
+
+#include "common/io/fdRead.h"
+#include "common/io/fdWrite.h"
+#include "common/fork.h"
+
+#include "common/harnessConfig.h"
+#include "common/harnessDebug.h"
+#include "common/harnessIo.h"
+#include "common/harnessLog.h"
+#include "common/harnessProtocol.h"
+#include "common/wait.h"
+#include "storage/storage.h"
+
```

Comment:
Sure seems like we have a lot of things include'd above- do we really need all of these?

Reply:
looks like I can reduce this to
  #include "common/harnessConfig.h"
  #include "common/harnessDebug.h"
  #include "common/harnessIo.h"
will run through tests to verify. I must have copied another file for template and failed to comment the headers out and add back in as needed.